x/distribution/types/params_test.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func TestValidateFuncNilSafety(t *testing.T) {
+	require.NotPanics(t, func() { validateCommunityTax(sdk.Dec{}) })
```

Comment:
I don't see any panics in the functions. Do you mean `require.NoError`?

Reply:
Good point. These should be `require.Error(...)`.

Reply:
Nope. It would panic

Reply:
It needs to be updated because it will no longer panic -- this is what the PR fixes.

Reply:
Ok will do